components/recorder-cli/mocha/index.test.mjs

```diff
@@ -45,18 +45,49 @@ setImmediate(() => {
   receiveMockSocket(getLastMockSocket(), "0");
 });
 
-await beforeEach.call({
-  currentTest: {
-    parent: {
-      fullTitle: () => "full-title-1",
+function startTest() {
```

Comment:
Arrow? I should have added some lint rules that flags functions when there is not a this expression in the body.